rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -145,10 +149,36 @@ def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False,
         # Image sizes of a batch
         self.img_size = np.zeros((self.bs * 2), dtype="int32")
 
+    def get_cupy_array_from_address(self):
+        if(self.tensor_format == types.NCHW):
+            address = self.loader.copyToHipTensorNCHW(
+                self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
+            print("Address : ",address)
+            mem = cp.cuda.memory.UnownedMemory(address, 0, None, self.device_id)
```

Comment:
Why the size is 0 here

Reply:
We are wrap our pointer using CuPy's UnownedMemory to get the memory pointer of type cupy.cuda.MemoryPointer required to create the cupy ndarray
The memptr points to the array content head 
The "mem" is not used anywhere except to get the pointer, hence size was given as "0"